examples/anchor_text_movie.ipynb

```diff
@@ -17,18 +17,28 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    %load_ext autoreload
</pre><p>I would remove these reload magics as there's no need for these in public examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
removed it